gcs/src/components/dashboard/map.jsx

```diff
@@ -21,7 +26,15 @@ import arrow from '../../assets/arrow.svg'
 import resolveConfig from 'tailwindcss/resolveConfig'
 import tailwindConfig from '../../../tailwind.config'
```

Comment:
can you move the stuff that isn't tailwind styling into its own import section here